gqcp/include/Basis/TransformationMatrix.hpp

```diff
@@ -56,6 +58,51 @@ class TransformationMatrix : public SquareMatrix<_TransformationScalar> {
 
         (*this) = (*this) * T;
     }
+
+    /**
+     *  @return returns this transformation as to the $VEC group format of GAMESS-US
+     */ 
+    std::string asGamessUsVecGroup() {
+        std::string output = "$VEC";
+
+        std::ostringstream doubleStream;  // Allows conversion of double to the right formatted string
+        // Format parameters for $VEC group file
+        doubleStream<<std::setprecision(8); 
+        doubleStream<<std::scientific;
+
+        for (size_t i = 0; i < this->dimension(); i++) {
+            for (size_t j = 0; j < this->dimension(); j++) {
+
+                // Every 5 coefficients is a new line
+                if (j % 5 == 0) {
+                    output += "\n";
+                    output += std::to_string((i+1));
+                    output += "  ";
+                    output += std::to_string(((j / 5) + 1));
+                }
+
+                double x = this->operator()(j ,i);
```

Comment:
```suggestion
                double x = this->operator()(j,i);
```

Does GAMESS-US have the convention that the coefficients of a new vector are found in a **row**, instead of in a column as we have?